entity_scenes/Player.gd

```diff
@@ -2,11 +2,11 @@ extends KinematicBody2D
 # The player's speed
```

Comment:
Would be nice if the Player extends AnimatedEntity.gd

Reply:
Yep, I think I added that to a later commit but haven't yet pushed it. I'll be sure to do that, so I guess we can leave this pull request open for the moment.